### PR TITLE
Adding updated content to README.md and remove superfluous sections 

### PR DESCRIPTION
**Pull request by ahopp**
### Description
* Add darkmode logo
* Add a ToC and relevant sections
* Add Project Resources to replace how can you help", "running tests", "guiding principles", etc.
* Add CoC
* Add License
* Add Copyright
 
### Issues Resolved
Partially Resolve: Issue #592

### CODE REVIEW

**Comment by ahopp** [comment]
Signed and added more descriptive title.

**Comment by kavilla** [comment]
I see the PR title (although 50+ characters) and description was updated nicely,

but the actual commit (e39b78a140b819fcd19e9d62fb31b5920d7cf556) will need to be modified. which you can do in your terminal, command prompt, or git bash in the following way:

* Get latest from main `git pull main`
* Do an interactive rebase `git rebase -i main`
You should see the following:
Screen Shot 2021-07-08 at 12 27 45 PM
* Tell git you want to reword the bad commit, so modify the rebase todo file to look like the following:
Screen Shot 2021-07-08 at 12 29 41 PM
Notice the change from 'pick' to 'r'
* Save (write out) and exit the file
* It should bring you back to the commit for rewording so now you can update the message, description, and sign it.
* Save (write out) and exit the commit
* `git push -f`

File: README.md [hunk 1]
```diff
@@ -1,78 +1,30 @@
-# OpenSearch Dashboards
+<img src="https://opensearch.org/assets/brand/SVG/Logo/opensearch_logo_default.svg" height="64px"/>
```
Comment (tmarkley) [inline README.md:1]: Can we use the Dashboards logo?

**Comment by tmarkley** [comment]
Need this to be deployed first to reference the Dashboards logo: https://github.com/opensearch-project/project-website/pull/224

Also I looked into dynamically rendering the default/dark mode logo based on the GitHub theme but this doesn't seem supported yet: https://github.community/t/support-theme-context-for-images-in-light-vs-dark-mode/147981/60

**Review by mihirsoni — APPROVED** [review]
LGTM !! Thanks for the changes.

**Review by kavilla — APPROVED** [review]

**Comment by tmarkley** [comment]
This for sure needs to go into v1.1, but should we push a patch v1.0.1 with this and other documentation changes included?

**Pull request merged**